app/models/space_indicator.rb

```diff
@@ -4,7 +4,7 @@ class SpaceIndicator < ApplicationRecord
 
   def self.create_indicators_for_space(space_indicators, space)
     indicators = []
-    space_indicators.each do |indicator|
+    (space_indicators || []).each do |indicator|
```

Comment:
I noticed without this there would be a 500 in user if this was nil

Reply:
Oh! Good catch!